framework/helpers/BaseHtml.php

```diff
@@ -754,7 +754,13 @@ protected static function booleanInput($type, $name, $checked = false, $options
      * the labels will also be HTML-encoded.
      * @param array $options the tag options in terms of name-value pairs. The following options are specially handled:
      *
-     * - prompt: string, a prompt text to be displayed as the first option;
+     * - prompt: string, a prompt text to be displayed as the first option. Since version 2.0.11 you can use array to
```

Comment:
Since version 2.0.11 you can use an array to override the value and to set other tag attributes: